### ISSUE TITLE
Training Cost

### ISSUE THREAD

**Issue by AmberCheng** [tumurzakov/AnimateDiff#21]
Hi, I am currently also fine-tuning the motion module of Animatediff. However, I am using the original GitHub（https://github.com/guoyww/AnimateDiff）. I found that when the training size is 512x512x16 frames, the GPU memory has reached 51G. I wonder what are your training configurations and resource consumption? Looking forward to your reply~

**Comment by tumurzakov** [comment]
Thats too many, 512x512x16 must took somewhere 15-16gb, I guess you didn't turn off gradients on unet. You must turn on grandients only on motion module.
